packages/react-core/src/helpers/util.ts

```diff
@@ -319,7 +319,7 @@ export const createRenderableFavorites = (
     return favoriteItems;
   }
   return (items as React.ReactElement[])
-    .filter(item => favorites.includes(item.props.id))
+    .filter(item => 'props' in item && favorites.includes(item.props.id))
```

Comment:
React.Element should have props and it's not correct to check this here. 
I answered in the issue you opened.